From 4845f1d6d8b0d9ab3213e2af6054cc8f1e65c4b8 Mon Sep 17 00:00:00 2001
From: Hans-Kristian Arntzen <post@arntzen-software.no>
Date: Thu, 12 Dec 2024 11:59:37 +0100
Subject: [PATCH 6/6] radv: Add radv_invariant_geom=true for Indiana Jones.

Water puddles expect invariant position, but does not declare such in
the vertex shaders, leading to random glitches.

Signed-off-by: Hans-Kristian Arntzen <post@arntzen-software.no>
Cc: mesa-stable
Reviewed-by: Samuel Pitoiset <samuel.pitoiset@gmail.com>
Part-of: <https://gitlab.freedesktop.org/mesa/mesa/-/merge_requests/32607>
(cherry picked from commit e815d6523c2baa1f7c80d8c25823ac249846bb13)
---
 src/util/00-radv-defaults.conf | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/util/00-radv-defaults.conf b/src/util/00-radv-defaults.conf
index d125e24ede8..be85f17453b 100644
--- a/src/util/00-radv-defaults.conf
+++ b/src/util/00-radv-defaults.conf
@@ -113,6 +113,7 @@ Application bugs worked around in this file:
             <option name="radv_zero_vram" value="true" />
             <option name="radv_legacy_sparse_binding" value="true" />
             <option name="radv_disable_dcc_stores" value="true" />
+            <option name="radv_invariant_geom" value="true" />
         </application>
 
         <application name="DOOM (2016)" application_name_match="DOOM$">
-- 
2.47.1

